docs/csharp/language-reference/compiler-options/langversion-compiler-option.md

```diff
@@ -61,7 +61,7 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
  
  While newer versions of the C# syntax do not explicitly depend on a specific version of the net framework runtime, compilers on older versions of the framework are not updated to support newer langauge syntax without additional configuration (which is beyond the scope of this article.) The above table lists the minimum net framework compiler version needed to compile for a given language version without any additional configuration needed.
  
- Additionally, C# 6 and newer are not supported at all with this compiler, they require migrating to the newer .NET Compiler Platform ("Roslyn")
+ Additionally, C# 6 and newer are not supported at all with the compiler included with the net framework as it is no longer being updated for features, they require migrating to the newer [.NET Compiler Platform ("Roslyn")](https://github.com/dotnet/roslyn)
```

Comment:
One more thing: the documentation should spell names properly, that is ".NET Framework" instead of "net framework".